### ISSUE TITLE
README.md is badly worded

### ISSUE THREAD

**Issue by jrigg** [linuxaudio/new-session-manager#67]
> It is a community version of the "NON Session Manager" and free in every sense of the word: free of cost, free to share and use, free of spyware or ads, free-and-open-source.

The above statement from README.md gives a misleading impression, as it implies that Non Session Manager is non-free and contains spyware and ads. I think it could be worded better to avoid confusion.

**Comment by dvzrv** [comment]
@jrigg Thanks for the report.

How would you rephrase it to remove the ambiguity? Suggestions welcome!

**Comment by falkTX** [comment]
Suggestions welcome.

Perhaps just separating the 2 sentences would be enough?

> It is a community version of the "NON Session Manager". It is free in every sense of the word: free of cost, free to share and use, free of spyware or ads, free-and-open-source.

**Comment by diovudau** [comment]
fixed in git.
